content/creator/scenes/releases/version-support-agreement.md

```diff
@@ -0,0 +1,69 @@
+---
+date: 2023-01-04
+title: Version support agreement
+description: Version support agreement
+categories:
+  - development-guide
+type: Document
+url: /creator/releases/releases
+weight: 2
+---
+
+
+This document describes the versioning policy applied to both the Decentraland Editor and the scripting framework (the `@dcl/sdk` package).
+
+The goal of this versioning policy is to forge a contract between the Decentraland Foundation's SDK team and content creators, to establish clear expectations and allow content creators to plan their work acoordingly.
+
+<!-- TODO: What about support from other teams? Do we have a commitment for content with older versions to keep working too? -->
+
+## Definitions
+
+* *Version number* -- a unique identifier for a publicly available version of software. This identifier consists of a *major version* number and a *minor version* number, separated dots (for example, 7.2).
+
+* *Version family* -- all versions that have identical major version form a family. We call version A a *successor* of version B if A and B belong to same family and A minor number is higher than B's (for example, 7.11 is a successor of 7.10).
+
+* *Breaking change* -- a change that forces a user to change their code or assets to keep them in functioning state. For example, a property changes name, and forces the user to change that property name every time it's used throughout their code.
+
+* *Non-breaking change* -- a change that does not require any action by a user, the behavior and properties of the user's code and assets are unchanged if they migrate their scene.
+
+* *Stable version* -- a version that forbids breaking changes in all its successors (which are all considered as stable too). Any breaking change must be introduced only through creation of a new version family by incrementing the major version number. Non-breaking changes are reflected through incrementing the minor version number.
```

Comment:
"Any breaking change must be introduced only through creation of a new version family by incrementing the major version number."

Perhaps add something like "however see caveat in ## breaking changes section".